src/bsp/src/stm32/hal/stm32f429zi/CMakeLists.txt

```diff
@@ -86,7 +111,9 @@ target_link_libraries(${LIB_NAME}
         HAL::STM32::F4::CORTEX
         STM32::NoSys
         SwarmUS::HiveMind::BSP::Common
-   
+
+        SwarmUS::HiveMind::FreeRTOS
```

Comment:
Does it need to be public? i.e: do you need any freertos symbols for api exposed to  the stm32 bsp?

Reply:
Moved to linker library as it is needed by CMSIS_RTOS